eng/pipelines/templates/stages/archetype-sdk-publish-net.yml

```diff
@@ -0,0 +1,204 @@
+parameters:
+  dependsOn: []
+  ShouldTag: true
+  ShouldSign: true
+  ShouldPublish: true
+  ShouldPublishSymbols: true
+  # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
+  DevOpsFeedId: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
+
+  ShouldPublishExecutables: false
+  RequireStrongNames: true
+  ExeMatrix: []
+  SigningPool:
+    name: azsdk-pool
+    image: windows-2022
+    os: windows
+
+# Needs Github Release variable group linked.
+# Needs AzureSDK Nuget Release Pipelines Secrets variable group linked.
+
+stages:
+- stage: Release
+  dependsOn: ${{ parameters.dependsOn }}
+  variables:
+    Artifacts: $(System.DefaultWorkingDirectory)/_artifacts/packages
```

Comment:
Do we want these in `$(Build.ArtifactStagingDirectory)` or is there any benefit/drawback to having them alongside source?

Reply:
I don't believe there is any benefit. I'll fix this in a follow-up PR that I address the create-tags-and-git-release in.